src/Item.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import Button from './Button';
+
+function Item({ todo, onClick }) {
```

Comment:
```suggestion
function Item({ todo: { content, id }, onClick }) {
```
한번 더 destructuring을 해줘도 좋겠네요~